design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -26,7 +26,9 @@ type Mutation {
     updateGiftRegistry(giftRegistryUid: ID!, giftRegistry: UpdateGiftRegistryInput!): UpdateGiftRegistryOutput
     removeGiftRegistry(giftRegistryUid: ID!): RemoveGiftRegistryOutput
 
-    addGiftRegistryItems(giftRegistryUid: ID!, items: [AddGiftRegistryItemInput!]!): AddGiftRegistryItemsOutput
+    addGiftRegistryItems(giftRegistryUid: ID!, items: [AddGiftRegistryItemInput!]!): AddGiftRegistryItemsOutput @deprecated(reason: "Use `moveCartItemsToGiftRegistry` instead. Not supported for gift registry") @doc(description: "Adds individual items to the gift registry")
```

Comment:
@cpartica could you please clarify why this mutation has been deprecated, and suggests to use the `moveCartItemsToGiftRegistry`, since the purpose of these 2 mutations are a bit different?

Reply:
@cpartica could you please remove the mutation from GQL schema? I've opened a PR that removes its implementation, and would be good to have the same changes here.
Thank you.